src/functions.c

```diff
@@ -370,6 +370,34 @@ void fcallCommandReadOnly(client *c) {
     fcallCommandGeneric(c, 1);
 }
 
+void functionsFlushCommand(client *c) {
```

Comment:
nit: shouldn't this be functionFlushCommand? I just realized there is an extra s on all the calls though, so not really something we need to fix.

Reply:
Right, will make another PR to fix it to all the function command callbacks.

Reply:
PR for the rest of the function callbacks: https://github.com/redis/redis/pull/9947
`functionsFlushCommand` will be renamed on this PR